common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -53,6 +53,8 @@ object Messages {
     namespace: Namespace,
     packageId: PackageId) extends BusMessage
 
+  final case class ImageStorageUsage(namespace: Namespace, timestamp: Instant, byteCount: Long) extends BusMessage
```

Comment:
Yes, but the others are wrong ;)

But seriously, the other message formatting was all changed in a recent commit for some reason and changed the whole formatting to something I would not consider better.